src/ServicePulse.Host/app/css/particular.css

```diff
@@ -1465,4 +1465,15 @@ span.metadata.danger, i.fa.fa-trash-o.danger, sp-moment.danger {
 
 .graph svg {
     width:100%;
+}
+
+.endpointrow {
```

Comment:
Can we agree on a convention for naming HTML classes? I've been separating multiple names in class with a hyphen to make it easier to read.